From 715115bc6ee28f71a585e63cc2416d26e026b873 Mon Sep 17 00:00:00 2001
From: Florinel Iordache <florinel.iordache@nxp.com>
Date: Mon, 5 Nov 2018 17:03:04 +0200
Subject: [PATCH 033/741] arm64: dts: lx2160: DPMAC connections to backplane
 PHYs example

commit 3663f09945b47a5598b7cbc88a5755aa8d1dac1c from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

This is an example of device tree nodes required to enable 10GBase-KR and 40GBase-KR on LX2160

Signed-off-by: Florinel Iordache <florinel.iordache@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 .../boot/dts/freescale/fsl-lx2160a-qds.dts    | 20 +++++++++++++++++++
 1 file changed, 20 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-lx2160a-qds.dts b/arch/arm64/boot/dts/freescale/fsl-lx2160a-qds.dts
index fed55d5c6c37..638e8e0d19f6 100644
--- a/arch/arm64/boot/dts/freescale/fsl-lx2160a-qds.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-lx2160a-qds.dts
@@ -202,3 +202,23 @@
 		fsl,lane-reg = <0xE00 0x100>; /* lane G */
 	};
 };
+
+/* Update DPMAC connections to 40G backplane PHYs
+ * &dpmac1 {
+ *	phy-handle = <&pcs_phy1>;
+ * };
+ * 
+ * &dpmac2 {
+ *	phy-handle = <&pcs_phy2>;
+ * };
+ */
+
+/* Update DPMAC connections to 10G backplane PHYs
+ * &dpmac3 {
+ *	phy-handle = <&pcs_phy3>;
+ * };
+ * 
+ * &dpmac4 {
+ *	phy-handle = <&pcs_phy4>;
+ * };
+ */
-- 
2.17.1

